test/Ed25519VerificationKey2020.spec.js

```diff
@@ -288,6 +288,24 @@ describe('Ed25519VerificationKey2020', () => {
       ).to.be.true;
     });
 
+    it('2020 key should round trip serialize to 2018', async () => {
+      const keyPair2020 = await Ed25519VerificationKey2020.generate({
+        seed: seedBytes, controller: 'did:example:1234'
+      });
+
+      const serialized2018 = keyPair2020.toEd255519VerificationKey2018({
+        publicKey: true, privateKey: true
+      });
+
+      const parsedKeyPair = await Ed25519VerificationKey2020
+        .from(serialized2018);
+
+      expect(keyPair2020.publicKeyMultibase).to
+        .equal(parsedKeyPair.publicKeyMultibase);
+      expect(keyPair2020.publicKeyMultibase).to
+        .equal(parsedKeyPair.publicKeyMultibase);
```

Comment:
It's great that this is comparing against itself, but could we also add a static test vector? If you don't have time for that, please file an issue for someone else to do it.

Reply:
Ah, no problem, I'll add a static vector while it's fresh in my mind